### PR TITLE
Fixes omnibox pedal icon.

### PR DESCRIPTION
**Pull request by mkarolin**
Replaces Chrome pedal icon with Brave lion.

Fixes brave/brave-browser#20223

## Submitter Checklist:

- [ ] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by mkarolin** [comment]
image

File: vector_icons/components/omnibox/browser/vector_icons/pedal.icon [hunk 1]
```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (simonhong) [inline vector_icons/components/omnibox/browser/vector_icons/pedal.icon:1]: nit: maybe our license sentence should be used?

**Review by simonhong — APPROVED** [review]

File: vector_icons/components/omnibox/browser/vector_icons/pedal.icon (hunk 1 above)
Comment (mkarolin) [reply vector_icons/components/omnibox/browser/vector_icons/pedal.icon]: Good catch. I copied the file from `product.icon` in the same directory and it has that copyright note. Updated in both. The rest of the files the directory have a different copyright (Mozilla), so I'll leave those as they are.

**Comment by mkarolin** [comment]
The other thing, is that the upstream `pedal.icon` has a canvas size 16, while the one I am replacing it with has size 32. It doesn't look bad, but it's better to avoid scaling. @bradleyrichter (not sure who else to ask), do we have an icon like this in size 16? (The upstream icon is https://source.chromium.org/chromium/chromium/src/+/main:components/omnibox/browser/vector_icons/pedal.icon).

**Comment by mkarolin** [comment]
> The other thing, is that the upstream `pedal.icon` has a canvas size 16, while the one I am replacing it with has size 32. It doesn't look bad, but it's better to avoid scaling. 
After discussing on Slack, leaving it at 32.

**Pull request merged**